styles/components/_formErrors.scss

```diff
@@ -0,0 +1,21 @@
+.errorList {
```

Comment:
We can wait for Mike to provide a design. Mostly you'd put some container styles on it.